UnityProject/Assets/Scripts/UI/Core/Action/UIActionManager.cs

```diff
@@ -186,6 +186,12 @@ public static void SetCooldown(IActionGUI iActionGUI, float cooldown, GameObject
 
 		public static void Show(IActionGUI iActionGUI)
 		{
+			foreach (var actionButton in Instance.DicIActionGUI)
+			{
+				//If there is a duplicate of this button, don't spawn it!
+				if (actionButton.Value.ActionData != iActionGUI.ActionData) continue;
```

Comment:
Isn't it simpler to invert this logic so no continue is needed?

Reply:
Done